yt_dlp/extractor/facebook.py

```diff
@@ -616,6 +603,30 @@ def parse_graphql_video(video):
                                 'url': playable_url,
                             })
                     extract_dash_manifest(video, formats)
+
+                    automatic_captions, subtitles = {}, {}
+                    is_broadcast = traverse_obj(video, ('is_video_broadcast', {bool}))
+                    for caption in traverse_obj(video, (
+                        'video_available_captions_locales',
+                        {lambda x: sorted(x, key=lambda c: c['locale'])},
+                        lambda _, v: url_or_none(v['captions_url'])
+                    )):
+                        lang = caption.get('localized_language', '')
+                        subs = {
+                            'url': caption['captions_url'],
+                            'name': format_field(caption, 'localized_country', f'{lang} (%s)', default=lang),
+                        }
+                        if caption.get('localized_creation_method') or is_broadcast:
+                            automatic_captions.setdefault(caption['locale'], []).append(subs)
+                        else:
+                            subtitles.setdefault(caption['locale'], []).append(subs)
+                    else:
+                        if not automatic_captions and not subtitles:
+                            if captions_url := traverse_obj(video, ('captions_url', {url_or_none})):
+                                locale = self._html_search_meta(
+                                    ['og:locale', 'twitter:locale'], webpage, 'locale', default='en_US')
+                                (automatic_captions if is_broadcast else subtitles)[locale] = [{'url': captions_url}]
```

Comment:
oops, else was not what we wanted here, mb
```suggestion
                    captions_url = traverse_obj(video, ('captions_url', {url_or_none}))
                    if captions_url and not automatic_captions and not subtitles:
                            locale = self._html_search_meta(
                                ['og:locale', 'twitter:locale'], webpage, 'locale', default='en_US')
                            (automatic_captions if is_broadcast else subtitles)[locale] = [{'url': captions_url}]
```